### ISSUE TITLE
Docker questions

### ISSUE THREAD

**Issue by hinto-janaiyo** [CryptoGrampy/HotShop#91]
The Docker build errors, but the container still runs great :D
```
hinto$[0] ~/git/HotShop sudo docker-compose up
[+] Running 0/1
 ⠿ hotshop Error          
```
Also, does this mean HotShop is running on all my threads? Is there a setting to limit it? 
I mean, a kiosk running on 32 threads is sick but I don't know if the code takes advantage of it
```
hotshop  | 2022/08/05 13:14:24 [notice] 1#1: start worker processes
hotshop  | 2022/08/05 13:14:24 [notice] 1#1: start worker process 7
...
hotshop  | 2022/08/05 13:14:24 [notice] 1#1: start worker process 38
```
I'm also wondering if the only thing HotShop connects to is a monerod, where does the fiat conversion come from?

P.S. the faded 0 is very slick

![nice](https://user-images.githubusercontent.com/101352116/183090608-b643e2eb-c7a3-4f45-ba1a-f5cd8f0f7ed9.jpg)

**Comment by CryptoGrampy** [comment]
Hi @hinto-janaiyo - thanks for testing this out.  Honestly, haven't used the docker build much since I wrote it- you're more than welcome to try and fix the error.  Not quite sure what would cause it, though... Probably related to the wasm files from monero-javascript.  Regarding multithread: https://github.com/monero-ecosystem/monero-javascript/issues/96

As for other things- I tried to be fairly explicit about what is happening under the hood in the 'About' section- you're correct that no connections are happening outside of the request for the static page assets (and no view keys get sent in this http request) and the direct browser -> web-compatible monerod request.  However!  You are correct that the exchange rate does come from the CoinGecko API- I will make a bug ticket to add this information to the About page as well as a note on the Settings page.  

Yes, I agree on the slickness of the 0 :) 

Looking forward to more feedback!
